Dashboard/Dashboard/Presentation/PrimaryCourseDashboardView.swift

```diff
@@ -277,7 +280,7 @@ public struct PrimaryCourseDashboardView<ProgramView: View>: View {
     }
     
     private func learnTitleAndSearch(proxy: GeometryProxy) -> some View {
-        let showDropdown = viewModel.config.program.enabled && viewModel.config.program.isWebViewConfigured
+        let showDropdown = true //viewModel.config.program.enabled && viewModel.config.program.isWebViewConfigured
```

Comment:
I believe this is an accidental comment. PLease revert it.

Reply:
Thanks! 🙏